rush.json

```diff
@@ -26,14 +26,12 @@
     {
       "packageName": "@uifabric/charting",
       "projectFolder": "packages/charting",
-      "versionPolicyName": "lockedMajor",
       "shouldPublish": false
     },
     {
       "packageName": "@uifabric/dashboard-grid-layout",
       "projectFolder": "packages/dashboard-grid-layout",
-      "versionPolicyName": "lockedMajor",
-      "shouldPublish": true
+      "shouldPublish": false
```

Comment:
>shouldPublish [](start = 7, length = 13)

My understanding is that "versionPolicyName" and "shouldPublish" are mutually exclusive.

"shouldPublish" is the old one, when we had only one model.  It defaults to **false** which makes sense for that model.

"versionPolicyName" is the new one.  In the future we will rename this to something more intuitive like "publishingModel" and there will be a predefined value called "none" which will mean "don't publish".  I'm pretty sure you do NOT need `"shouldPublish": true` when using a version policy, and I would recommend against it, because it's confusing (given the default of false).

Reply:
Is the call to action here to remove shouldPublish: true from everything else? I know @dzearing was relying on this flag in package.json for some of the beta publishing scripts...

Reply:
Thanks for taking a look so quickly 😄

Reply:
Can you provide a link to that code?  (BTW make sure not to confuse `"shouldPublish": true` in rush.json with `"private": false` in package.json)

Reply:
Hmmm....  This code should really be using the [@microsoft/rush-lib API](https://microsoft.github.io/web-build-tools/api/rush-lib.html).  `JSON.parse(fs.readFileSync('../rush.json', 'utf8'))` won't support JSON comments correctly, and if there is a syntax error, the error message won't be very informative.

Anyway your approach seems fine otherwise.

Reply:
Also are you sure this code is even used?  The `npm publish` line is commented out:

```javascript
for (const package of packages) {
  const packagePath = path.resolve('..', package.projectFolder);

  console.log(`Publishing ${chalk.magenta(package.packageName)} in ${packagePath}`);
  //  execSync('npm publish --tag beta', undefined, packagePath);
}
```

Reply:
It is, we manually uncomment that code when we need to actually publish.

Reply:
It isn't a big deal either way, I can make the change to remove all coexistence of shouldPublish and versionPolicy, or I could keep it this way to minimize amount of churn. :)